README.md

```diff
@@ -16,6 +16,7 @@
 </a>
 
 <br/><br/>
+
```

Comment:
Why?

Reply:
Better revert these changes, we don't want to do changes based on custom configs. If you want to enforce the format that should be done in a separate PR and CI should enforce it.

Reply:
Not all changes were reverted.

Reply:
Is there any easy way to see what changes are left now? I just tried to see through commit history but couldnt locate any more changes that need to be reverted.

Reply:
See the diff in https://github.com/zaproxy/zap-api-nodejs/pull/28/files

Reply:
What about now?